modules/microshift-config-parameters-table.adoc

```diff
@@ -65,27 +65,151 @@ The following table explains {microshift-short} configuration YAML parameters an
 |`number`
 |By default, `etcd` uses as much memory as needed to handle the load on the system. However, in memory constrained systems, it might be preferred or necessary to limit the amount of memory `etcd` can to use at a given time.
 
+|`ingress.defaultHTTPVersion`
+|`number`
+|Determines the default HTTP version to be used for ingress. Default value is `1`, which is the HTTP/1.1 protocol.
+
+|`ingress.forwardedHeaderPolicy`
+|`Append`, `Replace`, `IfNone`, `Never`
+|Specifies when and how the ingress router sets the `Forwarded`, `X-Forwarded-For`, `X-Forwarded-Host`, `X-Forwarded-Port`, `X-Forwarded-Proto`, and `X-Forwarded-Proto-Version` HTTP headers.
+
+`Append` specifies that the ingress router appends existing headers. `Append` is the default value.
+
+`Replace` specifies that the ingress router sets the headers and replaces any existing `Forwarded` or `X-Forwarded-*` headers.
+
+`IfNone` specifies that the ingress router sets headers if they are not already set.
+
+`Never` specifies that ingress router never sets the headers, preserving any existing headers.
+
+|`ingress.httpCompression`
+|`object`
+|`httpCompression` defines a policy for HTTP traffic compression. There is no HTTP compression by default.
+
+|`ingress.httpCompression.mimeTypes`
+|`array` or null
+|`mimeTypes` is a list of MIME types that should have compression applied. When the list is empty, the ingress controller does not apply compression. The format must follow the Content-Type definition in RFC 1341 that specifies the type and subtype of data in the body of a message and the native encoding of the data: `Content-Type := type \"/\" subtype *[\";\" parameter]`.
+
+The value of `Content-Type` can be one of the following types: application, audio, image, message, multipart, text, video, or a custom type preceded by `\"X-\"` and followed by a token. The token must be defined in one of the following ways:
+
+The token is a `string` of at least one character, and does not contain white spaces, control characters, or any of the characters in the `tspecials` set.
+
+The `tspecials` set contains the characters `()\u003c\u003e@,;:\\\"/[]?.=`.
+
+The subtype in Content-Type is also a token.
+
+The optional parameters following the subtype are defined as `token \"=\" (token / quoted-string)`.
+
+The `quoted-string`, as defined in RFC 822, is surrounded by double quotes and can contain white spaces plus any character except `\\`, `\"`, and `CR`. The `quoted-string` can also contain any single ASCII character if it is escaped by on the the following characters: `\\.",`.
+
+Not all MIME types benefit from compression, but `HAProxy` uses resources to try to compress if instructed to. Generally speaking, text formats such as `html`, `ccs`, and `js` benefit from compression. The benefits to files that are already compressed, such as images, audio, and video, are likely not worth the CPU resources spent compressing these a second time.
```

Comment:
I'm not sure where the "following ways" stop, but could they be bullets to make that clearer?

And note that if you do choose to use bullets, I believe you'll need to add an `a` (for the [AsciiDoc style](https://docs.asciidoctor.org/asciidoc/latest/tables/format-cell-content/)) before the beginning of the cell's `|` for them to work:

```suggestion
a|`mimeTypes` is a list of MIME types that should have compression applied. When the list is empty, the ingress controller does not apply compression. The format must follow the Content-Type definition in RFC 1341 that specifies the type and subtype of data in the body of a message and the native encoding of the data: `Content-Type := type \"/\" subtype *[\";\" parameter]`.

The value of `Content-Type` can be one of the following types: application, audio, image, message, multipart, text, video, or a custom type preceded by `\"X-\"` and followed by a token. The token must be defined in one of the following ways:

* The token is a `string` of at least one character, and does not contain white spaces, control characters, or any of the characters in the `tspecials` set.
* The `tspecials` set contains the characters `()\u003c\u003e@,;:\\\"/[]?.=`.
* The subtype in Content-Type is also a token.
* The optional parameters following the subtype are defined as `token \"=\" (token / quoted-string)`.
* The `quoted-string`, as defined in RFC 822, is surrounded by double quotes and can contain white spaces plus any character except `\\`, `\"`, and `CR`. The `quoted-string` can also contain any single ASCII character if it is escaped by on the the following characters: `\\.",`.

Not all MIME types benefit from compression, but `HAProxy` uses resources to try to compress if instructed to. Generally speaking, text formats such as `html`, `ccs`, and `js` benefit from compression. The benefits to files that are already compressed, such as images, audio, and video, are likely not worth the CPU resources spent compressing these a second time.
```